api/AudioListener.json

```diff
@@ -0,0 +1,849 @@
+{
+  "api": {
+    "AudioListener": {
```

Comment:
The usual changes:
- ordering of browsers
- an interface level compat and spec entry.
- no prefixing for property/method if it is at the interface level.